lib/cookie.js

```diff
@@ -1436,6 +1436,8 @@ CookieJar.prototype.removeAllCookies = function(cb) {
 
 CookieJar.prototype._cloneSync = syncWrap('clone');
 CookieJar.prototype.cloneSync = function(newStore) {
+  if(arguments.length === 0)
```

Comment:
Two syntactic details which would make this code change blend in even better: 

- even though this `if` has a 1-line body, use the `{` and `}` around it, just like the `if` just after this
- add a space after the `if` word – so that `if (arguments.length === 0) {`

Reply:
Recheck it